src/ProductBundle/Controller/ProductVariationAdminController.php

```diff
@@ -36,8 +43,15 @@ public function createAction(Request $request = null)
             throw new AccessDeniedException();
         }
 
+        // NEXT_MAJOR: Keep FQCN when bumping Symfony requirement to 2.8+.
+        if (method_exists(AbstractType::class, 'getBlockPrefix')) {
+            $integerType = IntegerType::class;
+        } else {
+            $integerType = 'integer';
+        }
+
         $form = $this->createFormBuilder(null, [])
-            ->add('number', 'integer', [
+            ->add('number', $integerType, [
```

Comment:
Remove the `if` and make this: `IntegerType::class`

Reply:
Okay, SF < 2.8 is dropped, so no more this stuff with `if` anywhere ;)

Reply:
I am preparing a PR to fully replace those, If you want, don't change this one